# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xtrans/hotfix.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xtrans-1.5.1/Xtrans.c.vanilla	2024-10-23 20:56:48.563222339 +0200
+++ xtrans-1.5.1/Xtrans.c	2024-10-23 20:57:39.929888274 +0200
@@ -149,6 +149,7 @@
 {
 #ifndef HAVE_STRCASECMP
     char 	protobuf[PROTOBUFSIZE];
+    int		i;
 #endif
 
     prmsg (3,"SelectTransport(%s)\n", protocol);
